client/flutter/assets/content_bundles/your_questions_answered.fr_FR.yaml

```diff
@@ -0,0 +1,221 @@
+
+        <p>🏠 <b>Si vous ne vous sentez pas bien, restez chez vous. En cas de fièvre, de toux et de difficultés respiratoires, consultez un médecin après l’avoir appelé. Suivez les instructions des autorités sanitaires locales.</b></p>
+
+        <p><i><b>Pourquoi ?</b> Ce sont les autorités nationales et locales qui disposent des informations les plus récentes sur la situation dans la région où vous vous trouvez. Si vous l&#39;appelez, votre médecin pourra vous orienter rapidement vers l’établissement de santé le plus adapté.  En outre, cela vous protègera et évitera la propagation de virus et d’autres agents infectieux.</i></p>
+
+        <p>🧳 <b>Informez-vous constamment pour savoir quelles sont les villes ou les zones où la COVID-19 se propage à grande échelle. Si possible, évitez de voyager, en particulier si vous êtes âgé ou diabétique ou si vous avez une maladie cardiaque ou pulmonaire.</b></p>
+
+        <p><i><b>Pourquoi ?</b> Le risque de contracter la COVID-19 est plus élevé dans ces régions.</i></p>
+    -
+      title_html: |
+        Que dois-je faire si j&#39;ai visité une zone où le COVID-19 se propage ?
+      body_html: |
+        <p>Si vous vous êtes récemment rendu(e) (au cours des 14 derniers jours) dans des régions où le COVID-19 se propage, suivez les instructions décrites à la question 15. ( Comment puis-je me protéger et éviter que la maladie ne se propage ? ») et suivez les conseils présentés ci-dessus ( Mesures de protection pour tous ).</p>
+
+        <p>🏠 <b>Si vous commencez à vous sentir mal, et même si vous ne présentez que des symptômes bénins (maux de tête, légère fièvre à partir de 37,3 °C et écoulement nasal modéré), restez chez vous jusqu&#39;à la guérison.</b> </p>
+
+        <p>Si vous avez absolument besoin que quelqu’un vienne vous ravitailler ou si vous devez sortir, par exemple pour acheter à manger, portez un masque pour éviter d’infecter d’autres personnes.</p>
+
+        <p><b><i>Pourquoi ?</i></b></p>
+
+        <p>Éviter d’entrer en contact avec d’autres personnes et de se rendre dans des établissements de santé permettra à ces établissements de fonctionner plus efficacement et vous protègera, ainsi que les autres personnes, de la COVID-19 et d’autres maladies virales.</p>
+
+        <p>🤒 <b>En cas de fièvre, de toux et de difficultés respiratoires, consultez un médecin sans tarder, car il peut s’agir d’une infection respiratoire ou d’une autre affection grave. Appelez votre médecin et indiquez-lui si vous avez récemment voyagé ou été en contact avec des voyageurs.</b></p>
+
+        <p><b><i>Pourquoi ?</i></b></p>
+
+        <p>Si vous l&#39;appelez, votre médecin pourra vous orienter rapidement vers l’établissement de santé le plus adapté.  En outre, cela vous protègera et évitera la propagation de la COVID-19 et d’autres maladies virales.</p>
```

Comment:
The second sentence mean " Calling your doctor will protect you against coronavirus and other viruses" We might want to change that sentence.

Reply:
I agree, but again, WHO approved translation, used in the bot.